src/explorer/trialApp/TrialAppTreeItem.ts

```diff
@@ -16,6 +16,19 @@ import { TrialAppClient } from './TrialAppClient';
 
 export class TrialAppTreeItem extends SiteTreeItemBase implements ISiteTreeItem {
     public static contextValue: string = 'trialApp';
+    private static _settingsToHide: string[] = [
```

Comment:
I wasn't sure where to put this. Is this good or should I put this somewhere more useful?

Reply:
I think it's fine in here, but I'm not sure if I'd attach it to the tree.  Maybe just as a constant in the file.  Also, I'd call it `filteredSettings`.  No real reason why, just feels like filter is the appropriate verb here to me.

Reply:
I like "settingsToHide" better. "filteredSettings" means the opposite to me, aka that would the the only ones we _show_, not _hide_. For example, that's how "azure.resourceFilter" works

Reply:
I agree with Eric, I like "settingsToHide" because it removes the ambiguity of "filter". I'm going to change it to "settingsToHide" here and in the other PR.